src/components/digitalSpecimen/tourSteps/AnnotateTourSteps.tsx

```diff
@@ -0,0 +1,242 @@
+            }));
+        } else if (nextIndex === 19) {
+            dispatch(setAnnotationWizardDummyAnnotation({
+                '@id': 'dummyAnnotation',
+                '@type': 'ods:Annotation',
+                "dcterms:identifier": 'dummyAnnotation',
+                'ods:fdoType': 'https://doi.org/21.T11148/cf458ca9ee1d44a5608f',
+                'ods:status': 'Active',
+                'ods:version': 1,
+                'oa:motivation': 'ods:adding',
+                'oa:motivatedBy': '',
+                'oa:hasTarget': {
+                    '@id': 'dummyAnnotationTarget',
+                    '@type': 'ods:DigitalSpecimen',
+                    'dcterms:identifier': 'dummyAnnotationTarget',
+                    'ods:fdoType': 'https://doi.org/21.T11148/894b1e6cad57e921764e',
+                    'oa:hasSelector': {
+                        '@type': 'ods:ClassSelector',
+                        'ods:class': "$['ods:hasEntityRelationships']"
+                    }
+                },
+                'oa:hasBody': {
+                    '@type': 'oa:TextualBody',
+                    'oa:value': [
+                        '{"dwc:relationshipOfResource":"GeoCASe","ods:relatedResourceURI":"https://geocase.eu/specimen/GIT338-118"}'
+                    ]
+                },
+                'dcterms:creator': {
+                    '@id': 'dummyAnnotationCreator',
+                    '@type': 'prov:SoftwareAgent',
+                    'schema:identifier': 'dummyAnnotationCreator',
+                    'schema:name': 'Tour Manager'
+                },
+                'dcterms:created': '2024-11-15T08:56:50.758Z',
+                'dcterms:modified': '2024-11-15T08:56:50.758Z',
+                'dcterms:issued': '2024-11-15T08:56:50.758Z',
+                'as:generator': {
+                    '@id': 'dummyAnnotationGenerator',
+                    '@type': 'prov:SoftwareAgent',
+                    'schema:identifier': 'dummyAnnotationGenerator'
+                }
+            }));
+        }
+
+        if (nextIndex < 9 && nextIndex > 18) {
+            dispatch(setAnnotationWizardFormValues({
+                class: undefined,
+                annotationValues: {},
+                motivation: '',
+                jsonPath: ''
+            }));
+        }
+
+        if ([7, 8, 9].includes(nextIndex)) {
+            dispatch(setAnnotationWizardSelectedIndex(0));
+            dispatch(setAnnotationTarget(undefined));
+        } else if ([10, 11, 12].includes(nextIndex)) {
+            dispatch(setAnnotationWizardSelectedIndex(1));
+        } else if ([13, 14, 15, 16].includes(nextIndex)) {
+            dispatch(setAnnotationWizardSelectedIndex(2));
+        } else if ([17, 18].includes(nextIndex)) {
+            dispatch(setAnnotationWizardSelectedIndex(3));
+        }
+
+        if (nextIndex !== 19) {
+            dispatch(setAnnotationWizardDummyAnnotation(undefined));
+        }
+
+        setTimeout(() => {
```

Comment:
https://react.dev/reference/react/useEffect would this be helpful instead of a timeout?

Reply:
UseEffect can only be used in React Components, not in functions. This is a general rule for hooks: https://react.dev/reference/rules/rules-of-hooks.